sonar-scanner-engine/src/main/java/org/sonar/scanner/scan/ProjectLock.java

```diff
@@ -28,11 +28,16 @@
 import org.sonar.home.cache.DirectoryLock;
 import org.sonar.scanner.bootstrap.Slf4jLogger;
 
+import org.picocontainer.Startable;
+import org.sonar.api.batch.fs.internal.InputModuleHierarchy;
+import org.sonar.home.cache.DirectoryLock;
+import org.sonar.scanner.bootstrap.Slf4jLogger;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this duplicated import. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)